code/core/write_sdl_ntuple.cc

```diff
@@ -181,6 +181,30 @@ void createGnnNtupleBranches()
     ana.tx->createBranch<vector<float>>("LS_sim_vz");
     ana.tx->createBranch<vector<int>>("LS_isInTrueTC");
 
+    // T3 branches
+    ana.tx->createBranch<vector<int>>("t5_t3_idx0");
+    ana.tx->createBranch<vector<int>>("t5_t3_idx1");
```

Comment:
parallelism is broken here.
Technically these are t5 branches.
The comment //T3 branches showing right above this is misleading

Reply:
What do you mean by "parallelism is broken"? The comment is a typo and will be fixed with the responses to Manos's comments.